test/test_pebble.py

```diff
@@ -1955,12 +1958,12 @@ def test_pull_path_error(self):
         ])
 
     def test_pull_protocol_errors(self):
-        self.client.responses.append(({'Content-Type': 'ct'}, b''))
+        self.client.responses.append(({'Content-Type': 'c/t'}, b''))
         with self.assertRaises(pebble.ProtocolError) as cm:
             self.client.pull('/etc/hosts')
         self.assertIsInstance(cm.exception, pebble.Error)
         self.assertEqual(str(cm.exception),
-                         "expected Content-Type 'multipart/form-data', got 'ct'")
+                         "expected Content-Type 'multipart/form-data', got 'c/t'")
```

Comment:
Yeah, the test started failing without it, because the new `email.message.Message` header parsing validates that the content type is of the form `type/subtype`.